index.html

```diff
@@ -87,6 +87,80 @@ <h2 id="use-context">
   </div>
 </article>
 
+<article>
+  <h2 id="short-description">
+    <a href="#short-description">
+      Short description 
+    </a>
+  </h2>
+
+  <p>A spec description should never be longer than 40 characters. If this happens you should split it using a context.</p>
```

Comment:
🤔 I think 40 chars is too little isn't it? I can find a bunch of examples where a longer `it` description seems fine.
I suggest 100 chars just for the description.

Reply:
Yes, you are right.

Reply:
Addressed - [456842d](https://github.com/evenbetterspecs/evenbetterspecs.github.io/pull/1/commits/456842dbb99cdf68f60cc7323794fd4716122f60)